cocos/gfx/webgl/webgl-command-buffer.ts

```diff
@@ -126,13 +127,13 @@ export class WebGLCommandBuffer extends CommandBuffer {
         }
     }
 
-    public bindInputAssembler (inputAssembler: InputAssembler): void {
-        const gpuInputAssembler = (inputAssembler as WebGLInputAssembler).gpuInputAssembler;
+    public override bindInputAssembler (inputAssembler: InputAssembler): void {
+        const gpuInputAssembler = (inputAssembler as WebGLInputAssembler).getGpuInputAssembler();
```

Comment:
getter/setter could not be mangled since babel will convert it to `_createClass({})` with normal string which could not be mangled by terser.

Some reference: https://github.com/terser/terser/issues/322